kubernetes/dashboard-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cxc-dashboard
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: cxc-dashboard
+  template:
+    metadata:
+      labels:
+        app: cxc-dashboard
+    spec:
+      containers:
+        - name: cxc-dashboard
+          image: iotaledger/cxc-dashboard:v0.0.1
```

Comment:
you most likely need to add the command so it is able to run the dashboard as done for the is-dashboard:
`command: ['sh', '-c', 'npm run build && npm run prod']`